src/raftstore/store/worker/region.rs

```diff
@@ -333,6 +339,32 @@ impl SnapContext {
         Ok(())
     }
 
+    // check the number of files at level 0 to avoid write stall after ingesting sst,
+    // and return the number of files can to be ingested.
+    fn check_level_0_num_files(&self) -> u64 {
```

Comment:
The function name doesn't match its behavior.

Reply:
Do you have any good suggestion? I can't come up with a better name.